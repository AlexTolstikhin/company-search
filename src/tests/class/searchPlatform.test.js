import React from 'react';
import renderer from 'react-test-renderer';
import SearchPlatform from '../../components/class/searchPlatform';

it('snaps SearchPlatform with undefined props', () => {
	const tree = renderer
		.create(<SearchPlatform />)
		.toJSON();
	expect(tree).toMatchSnapshot();
});