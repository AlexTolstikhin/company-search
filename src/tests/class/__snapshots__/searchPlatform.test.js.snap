// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snaps SearchPlatform with undefined props 1`] = `
<div>
  <div
    style={
      Object {
        "alignItems": "center",
        "display": "flex",
        "height": "200px",
        "justifyContent": "center",
      }
    }
  >
    <form
      onSubmit={[Function]}
    >
      <input
        data-testid="input-element"
        onChange={[Function]}
        style={
          Object {
            "borderRadius": "15px",
            "fontSize": "15px",
            "height": "30px",
            "paddingLeft": "10px",
            "width": "600px",
          }
        }
        type="text"
        value=""
      />
      <button
        style={
          Object {
            "borderRadius": "15px",
            "cursor": "pointer",
            "fontWeight": 700,
            "height": "35px",
            "margin": "15px",
            "width": "100px",
          }
        }
      >
        Search...
      </button>
    </form>
  </div>
  <div
    style={
      Object {
        "display": "flex",
        "flexWrap": "wrap",
        "padding": "0 100px",
      }
    }
  />
</div>
`;
